docs/4___co-simulation.adoc

```diff
@@ -4,7 +4,7 @@ FMI for Co-Simulation provides a standardized interface for execution of simulat
 In contrast to FMI for Model Exchange, a Co-Simulation FMU implements not only the model algorithm, but also the required solution method.
 The data exchange between FMUs is restricted to discrete communication points latexmath:[\mathbf{t}_i] (unless <<IntermediateUpdateMode>> is used).
 In the time between two communication points (or between entering <<IntermediateUpdateMode>>), the subsystem inside an FMU is solved independently by internal means.
-This leads to a delay for information "traveling through" the FMU, so there cannot be an immediate reaction (direct feedthrough) as in <<InitializationMode>>, <<EventMode>> or <<ContinuousTimeMode>>.
+This leads to a delay for information "traveling through" the FMU, so there cannot be an immediate reaction (direct feedthrough) as in <<InitializationMode>>, <<EventMode>> or <<ContinuousTimeMode>> of Model Exchange.
```

Comment:
This is not the case in event mode, right? Should we state this explicitely?

Reply:
Line 7 lists the modes without delay capabilities. I am not sure what you are missing here...

Reply:
You are right. Please ignore my comment.